packages/node_modules/@webex/internal-plugin-devices/test/integration/spec/devices.js

```diff
@@ -472,7 +914,97 @@ describe('plugin-devices', () => {
       });
     });
 
-    describe('#unregister()', () => {});
-    describe('#waitForRegistration()', () => {});
+    describe('#unregister()', () => {
+      describe('when the device is registered', () => {
+        beforeEach('register the device', () => devices.register());
+
+        describe('when the unregistration request is sent', () => {
+          let url;
+
+          beforeEach('setup the \'request()\' stub', () => {
+            devices.request = sinon.stub().resolves();
+            url = devices.url;
+          });
+
+          it('should use the device\'s url property', () => {
+            assert.isTrue(devices.registered);
+
+            return devices.unregister()
+              .then(() =>
+                assert.calledWith(devices.request, sinon.match.has(
+                  'uri', url
+                )));
+          });
+
+          it('should send a DELETE request', () => {
+            assert.isTrue(devices.registered);
+
+            return devices.unregister()
+              .then(() =>
+                assert.calledWith(devices.request, sinon.match.has(
+                  'method', 'DELETE'
+                )));
+          });
+        });
+
+        describe('when the device unregistration request is successful', () => {
+          it('should clear the device properties', () => devices.unregister()
```

Comment:
```suggestion
          it('should clear the device url', () => devices.unregister()
```

Reply:
Changed this to match the scope of the `it()` method.